js/src/sdk/utils/telemetry/index.ts

```diff
@@ -32,12 +32,15 @@ export class TELEMETRY_LOGGER {
   private static async sendTelemetry(payload: Record<string, unknown>[]) {
     const isTelementryDisabled =
```

Comment:
There's a typo in the variable name 'isTelementryDisabled'. It should be 'isTelemetryDisabled'. This could cause confusion in the future.